src/tests/components/LocationDetails.test.js

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import LocationDetails from "../../components/LocationDetails";
+
+describe("LocationDetails", () => {
+  it("renders the correct city and location props", () => {
+    const { getByText } = render(<LocationDetails city="Leeds" country="UK" />);
+    expect(getByText("Leeds, UK")).toBeInstanceOf(HTMLHeadingElement);
```

Comment:
Its common convention to leave a clear line before the assertions start in each test to help with readability, should add one in here
```
    const { getByText } = render(<LocationDetails city="Leeds" country="UK" />);

    expect(getByText("Leeds, UK")).toBeInstanceOf(HTMLHeadingElement);
```